firebase-appdistribution/src/main/res/values/strings.xml

```diff
@@ -13,40 +13,37 @@
 <!-- limitations under the License. -->
 
 <resources>
-  <string name="signin_dialog_title">Enable testing features</string>
-  <string name="singin_dialog_message">Enable testing features like new build alerts and in-app feedback.</string>
-  <string name="singin_yes_button">Turn on</string>
-  <string name="singin_no_button">Not now</string>
-  <string name="no_update_available">No new release available, try calling checkForUpdate.</string>
-  <string name="update_dialog_title">New Version Available</string>
-  <string name="update_yes_button">Update</string>
-  <string name="update_no_button">Cancel</string>
-  <string name="downloading_app_update">Downloading in-app update…</string>
-  <string name="download_completed">Download completed</string>
-  <string name="download_failed">Download failed</string>
-  <string name="download_pending">Download pending</string>
-  <string name="install_failed">Install failed</string>
-  <string name="install_canceled">Install canceled</string>
-  <string name="update_canceled">Update canceled </string>
-  <string name="notifications_group_name">Firebase App Distribution</string>
-  <string name="app_update_notification_channel_name">App updates</string>
-  <string name="app_update_notification_channel_description">Shows progress of in-app updates</string>
-  <string name="feedback_launch_failed">Failed to launch feedback</string>
-  <string name="feedback_unidentified_release">Release not found. This app may not have been installed by App Distribution, or you may no longer have access.</string>
-  <string name="feedback_no_release">Would have sent feedback, but did not due to development mode.</string>
```

Comment:
Is this due to the Developer Mode issue? Is there a way to rephrase this such that the user knows what happened and how to resolve the issue?